server/src/routes/router.js

```diff
@@ -41,4 +42,10 @@ export default (app, express) => {
   //////////////////////////////////////////////
   app.post('/api/tags/job', tagController.addJobTags);
 
+  //////////////////////////////////////////////
+  //Handling Error
+  //////////////////////////////////////////////
+  app.get('*', (req, res)=>{
```

Comment:
so, this is a server side route -- I don't think this will work like we expect it to. I remember stephen grider mentioning something like when the route is changed, there isn't actually a get request to that route, the only thing that changes is the component in the SPA